src/pool/srv_target.c

```diff
@@ -916,75 +916,63 @@ ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 {
```

Comment:
It seems to me that if 'buf == NULL', this function will do nothing, so I'm wondering why do we bother to call this function with 'buf == NULL'? Should we assert(buf != NULL) in this function?

Reply:
If the buf is NULL, it will only update the sp_map_version and pool map version on each xstream.